jenkins/job-configs/kubernetes-jenkins-pull/charts-pull.yaml

```diff
@@ -1,5 +1,11 @@
+- project:
+    name: charts-pull
+    repo-name: 'kubernetes/charts'
+    jobs:
+        - '{name}-test-e2e'
+
 - job-template:
-    name: 'charts-pull-test-e2e'
+    name: '{name}-test-e2e'
     # defaults
     success-comment: ''
```

Comment:
to reiterate: unless you expect to have multiple PR jobs, I'd probably just collapse all the templating down into a single job (and remove unused bits like this)